tracer/src/Datadog.Trace.Tools.Runner/Program.cs

```diff
@@ -148,6 +148,10 @@ private static void ConfigureApp(IConfigurator config, ApplicationContext applic
                   .WithDescription("Apply AOT automatic instrumentation on application folder")
                   .WithExample("apply-aot c:\\input\\ c:\\output\\".Split(' '))
                   .IsHidden();
+
+            config.AddCommand<AnalyzeInstrumentationErrorsCommand>("analyze-instrumentation")
+                  .WithDescription("Analyze instrumentation errors")
+                  .WithExample("analyze-instrumentation [--name dotnet] [--pid 123456] [--log-path c:\\logs\\]".Split(' '));
```

Comment:
Should we consider changing `--name` to `--process-name`? I know it's longer but seems more precise?

Reply:
Fixed in d6e34e581c70cc2271806c12db2c3d4f2caf78f1